airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/WriteConfig.java

```diff
@@ -56,12 +66,4 @@ public SyncMode getSyncMode() {
     return syncMode;
   }
 
-  public boolean getTransactionMode() {
-    return transactionMode;
-  }
-
-  public void setTransactionMode(boolean transactionMode) {
```

Comment:
not used? what's the use case for this?

Reply:
IIRC Snowflake, Postgres, Redshift, and CSV are respecting Transactions when committing the final results to destination tables but BigQuery is not.

Sometimes, transactions are desirable, sometimes it would be nice to turn it off and replicate as much as possible, ignoring errors as reported in #1199 

This boolean was introduced to try to toggle that behavior on and off but might end up not being used for the moment...

Reply:
okay. yeah. let's wait until we need it to re-add it.